nav2_system_tests/src/error_codes/error_code_param.yaml

```diff
@@ -173,7 +173,7 @@ planner_server:
       plugin: "nav2_system_tests::NoValidPathErrorPlanner"
     no_viapoints_given:
       plugin: "nav2_system_tests::NoViapointsGivenErrorPlanner"
-    cancelled:
+    cancelled:  # codespell:ignore cancelled
```

Comment:
his needs to be updated since you changed the value in `palnner_plugins`. This is a test, so easy enough. Just change here to `canceled` and remove the codespell ignore